src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1966,7 +1972,9 @@ public void GetPartsOfCastExpression(SyntaxNode node, out SyntaxNode type, out S
             return null;
         }
 
-        public SyntaxNode GetContainingPropertyDeclaration(SyntaxNode node) => node.GetAncestor<PropertyDeclarationSyntax>();
+        public SyntaxNode GetContainingPropertyDeclaration(SyntaxNode node) => node.GetAncestorOrThis<PropertyDeclarationSyntax>();
+
+        public SyntaxNode GetContainingParameter(SyntaxNode node) => node.GetAncestorOrThis<ParameterSyntax>();
```

Comment:
for 'meaty' methods (i.e. more than a trivial `=> null` or `=> false`) can you wrap and indent prior to the `=>`?  This is also helpful on github where things get cutoff when the line is too long.